ietf/templates/iab_base.html

```diff
@@ -62,7 +54,7 @@
         {% block main_content %}
         <main id="content">
             <div class="bg-white border-bottom pb-3">
-                <div class="container">
+                <div class="container container--iab">
```

Comment:
`container--iab` no longer does anything (it's the custom screen widths we decided to discard). It's crept back in in a few places. It doesn't look like it's bought any other incorrect code with it and it can be removed anywhere you see it.

Reply:
Done - I have removed it from all templates not just this one.